docs/UserGuide.md

```diff
@@ -244,21 +244,21 @@ Format: `group [id/NUSID] [g/GROUP] [t/TAG]`
 
 Schedule a meeting with a person in the address book.
 
-Format: `schedule id/NUSID [s/SCHEDULE] [r/REMARK]`
+Format: `schedule id/NUSID [s/SCHEDULE r/REMARK]`
```

Comment:
Fix consistency that schedule and remark must be either both present or both absent.